wallettool/src/test/java/org/bitcoinj/wallettool/WalletToolTest.java

```diff
@@ -0,0 +1,49 @@
+package org.bitcoinj.wallettool;
```

Comment:
This file is missing the license.

Reply:
Fixed: 1d047ffba2012de71173b7baf8710f1e9444fed9